WcaOnRails/app/models/competition.rb

```diff
@@ -574,6 +588,10 @@ def events_with_rounds_with_results
       end
   end
 
+  def delegate_report
+    DelegateReport.find_by_competition_id(self.id) || DelegateReport.create!(competition_id: self.id, content: "Hey, enter some text here", posted_at: nil)
```

Comment:
Calling `delegate_report` here causes an infinite recursion. Calling `super` here doesn't work because we're not overriding a method defined in a parent class, this is a method that was defined by some metaprograming madness in ActiveRecord. We could give this method a different name, but it seems kind of silly to do so, since we're trying to simulate a world where `DelegateReport` *always* exists.